packages/fxa-settings/e2e/functional/settings.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const intern = require('intern').default;
+const { describe, it, beforeEach } = intern.getPlugin('interface.bdd');
+const FunctionalHelpers = require('../../../fxa-content-server/tests/functional/lib/helpers');
```

Comment:
I think we'd need to add `fxa-content-server` as a dependency to settings for this to work in CI. Maybe we want to create our own helper file for settings and only pull in helpers when we need to use them instead of all of them? Could give us an opportunity to clean them up to use async/await too and only use what we need, if we took that approach (since we'll _eventually_ get around to the signin/registration flow) we'll have a nice cleaned up helper file by the end of it.

Reply:
I'm for creating our own helper file and copying over what we need when we need it; there are a _lot_ of helpers in there and I am certain that there are some where cleaning them up or rewriting them altogether could be beneficial. It also allows us to make Intern upgrades to keep fxa-settings modern without breaking anything in content-server.